Draft.md

```diff
@@ -192,128 +197,130 @@ Die folgenden Fragen nehmen Bezug auf technische Merkmale, welche die Nutzererfa
 
-* **Gibt es eine Programmierschnittstelle (API)?** Neben der Bedienbarkeit durch einen Menschen, ist auch eine maschinelle Ansteuerung der Software wichtig. Nur so können beispielsweise komplexe Prozesse mit mehreren Softwarekomponenten komplett automatisiert werden. Eine API gewährleistet dies. Sie sollte möglichst offen und ggf. mittels eines Standards wie [OpenAPI](https://www.openapis.org) dokumentiert sein. Ein  Beispiel für APIs sind die Schnittstellen von Wikidata, an die man automatisierte Datenabfragen stellen kann.
+* **Gibt es eine Programmierschnittstelle (API)?** Neben der Bedienbarkeit durch einen Menschen ist auch eine maschinelle Ansteuerung der Software wichtig. Nur so können beispielsweise komplexe Prozesse mit mehreren Softwarekomponenten komplett automatisiert werden. Eine API gewährleistet dies. Sie sollte möglichst offen und ggf. mittels eines Standards wie [OpenAPI](https://www.openapis.org) (https://www.openapis.org [23.10.2020]) dokumentiert sein. Ein  Beispiel für APIs sind die Schnittstellen von Wikidata, an die man automatisierte Datenabfragen stellen kann.
 
 #### Konformität mit Regelungen zum Datenschutz und der Datensparsamkeit
 
 Für den Einsatz in der universitären Forschung und der Lehre entscheidet die Frage nach dem Datenschutz vielfach darüber, ob die Software überhaupt genutzt werden darf. Eine Einschätzung der Regelungen, sofern sie sich nicht eindeutig auf die europäischen Rahmenrichtlinien beziehen, ist teilweise jedoch kaum möglich und Gegenstand von juristischen Diskussionen. Auch hier kann die Rezension bereits durch den Hinweis auf das Thema einen wichtigen Service für die Leser bringen. Dies gilt auch für Datensparsamkeit und das Hinterfragen von Registrierungsvorgängen, Cookies und Ähnlichem.
 
-* **Werden die Gesetze (z. B. zu Datenschutz, Kartendarstellungen etc.) des Einsatzlandes durch die Software eingehalten?** Es muss damit gerechnet werden, dass die Software in verschiedenen Regionen und Ländern verwendet wird,  welche jeweils spezifische Gesetze haben, die sich auf die  Ausführung/Installation der Software auswirken. Ein Beispiel sind Grenzen auf Kartendarstellungen, die wegen Grenzkonflikten wie etwa der um Kashmir, zu unterschiedlichen Darstellungen in Indien, Pakistan, China und den übrigen Staaten führt. Ein weiteres Beispiel ist die EU-weite Datenschutzgrundverordnung.
+* **Werden die Gesetze (z. B. zu Datenschutz, Kartendarstellungen etc.) des Einsatzlandes durch die Software eingehalten?** Es muss damit gerechnet werden, dass die Software in verschiedenen Regionen und Ländern verwendet wird, welche jeweils spezifische Gesetze haben, die sich auf die Ausführung/Installation der Software auswirken. Ein Beispiel sind Grenzen auf Kartendarstellungen, die wegen Grenzkonflikten wie etwa der um Kaschmir zu unterschiedlichen Darstellungen in Indien, Pakistan, China und den übrigen Staaten führt. Ein weiteres Beispiel ist die EU-weit gültige Datenschutzgrundverordnung.
```

Comment:
> es wird für den/die Rezensierende*n schwierig sein, voraus zu sehen, in welchem Land und damit in welchem rechtlichen Kontext die Software eingesetzt wird. Vielleicht ist das Beispiel nur schlecht gewählt!?

Reply:
Das stimmt natürlich. Vielleicht muss es also auch an dieser Stelle darum gehen, dass eine Rezension dafür sensibilisieren kann, dass in einem typischen Ensatzbereich der Software, entsprechende Probleme auftreten können. Und dass das ein wichtiges Thema ist, zeigen doch gerade die leidigen Diskussionen rund um die verschiedenen Videokonferenzsystem ("Dürfen wir XY an unserer Uni nutzen? Ja, nein, vielleicht")

Reply:
Vielleicht ist es besser, das Kaschmir-Beispiel auszulassen. Stattdessen vielleicht nach *Ausführung/Installation der Software auswirken.*:

> Ein Beispiel sind Cloud-Anwendungen, die für einen Nordamerikanisches Nutzerkreis entwickelt wurden und mit der EU-weit gültigen Datenschutzgrundverordnung nicht kompatibel sind.

Reply:
So gelöst